php/php.phpunit/src/org/netbeans/modules/php/phpunit/preferences/PhpUnitPreferencesValidator.java

```diff
@@ -60,11 +60,12 @@ public PhpUnitPreferencesValidator validateCustomSuite(boolean customSuiteEnable
     }
 
     public PhpUnitPreferencesValidator validatePhpUnit(boolean phpUnitEnabled, String phpUnitPath) {
-        if (phpUnitEnabled) {
```

Comment:
Why this change in this file? It seems to me to be unchanged; or am I missing something?

Reply:
You are totally right, this has no change in functionality, only that it now has an early return if it's not enabled (like in [validatePath](https://github.com/apache/netbeans/blob/0cbb332843ef9c48452ae82c934df06f5523a6ce/php/php.phpunit/src/org/netbeans/modules/php/phpunit/preferences/PhpUnitPreferencesValidator.java#L73) ) instead of a nested positive case.
It is not relevant for this feature, I'm okay with removing this change, It is just a remnant of me trying to figure out a solution for my problem :)

Reply:
Personally, I prefer a PR to contain only changes related to the given subject. If we want this particular change, it is totally fine - create a tech-debt ticket and a separate PR, ideally containing more changes like this.

Reply:
Understood, makes perfect sense. I removed these changes and put everything into one commit.